create_aio_app/utils/generator.py

```diff
@@ -1,18 +1,12 @@
-from distutils.dir_util import copy_tree
-import shutil
 import os
+import shutil
+from distutils.dir_util import copy_tree
+
+from jinja2 import (BaseLoader, Environment)
```

Comment:
I prefer the old style for `import`. Can u reformat it?

```python
from jinja2 import (
    BaseLoader,
    Environment,
)
```

This style helpful if u watch git diff